resources/views/multiplayer/rooms/show.blade.php

```diff
@@ -18,7 +18,7 @@
 @section('ranking-header')
     <div class="osu-page osu-page--ranking-info">
         <div class="js-react--basic-select-options">
-            <div class="select-options select-options--spotlight">
```

Comment:
remove the style as well? and somehow I put this modifier for daily challenge blade view 🤨

Reply:
what the.... I thought I removed this already 🤨

Reply:
oh wait the style has been gone since forever, I was looking at different thing.